vector/src/main/res/layout/activity_medias_previewer.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/riot_primary_background_color_black">
+
+    <ImageView
+        android:id="@+id/image_previewer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerInParent="true"
+        android:scaleType="fitCenter"
+        android:visibility="gone"/>
+
+    <VideoView
+        android:id="@+id/video_previewer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerInParent="true"
+        android:scaleType="fitCenter"
+        android:visibility="gone"/>
+
+    <WebView
+        android:id="@+id/web_previewer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        android:visibility="gone"/>
+
+    <ImageView
+        android:id="@+id/file_previewer"
+        android:layout_width="265dip"
+        android:layout_height="265dip"
```

Comment:
Quality of the icon is not good.
Please use wrap_content for width and height and add
`android:src="@drawable/filetype_attachment"`
No need to set the src it in code then